mongodbatlas/peers.go

```diff
@@ -13,7 +13,7 @@ const peersPath = "groups/%s/peers"
 // endpoints of the MongoDB Atlas API.
 // See more: https://docs.atlas.mongodb.com/reference/api/peers/
```

Comment:
I have a small drive by fix here if you can address this here as well please
```suggestion
// See more: https://docs.atlas.mongodb.com/reference/api/vpc/
```